aries-site/src/examples/templates/feedback/PowerOnExample.js

```diff
@@ -0,0 +1,166 @@
+      });
+  }, []);
+
+  return (
+    <Card width="large" gap="medium" justify="center" pad="medium">
+      <Box>
+        <Heading level={1} size="small" margin="none">
+          Power On
+        </Heading>
+        <Anchor href="#" label="Learn more about server actions" />
+      </Box>
+      {!isSuccessful ? (
+        <Text size="large">You are powering on 1 device</Text>
+      ) : (
+        <Box align="center" gap="xsmall" direction="row">
+          <StatusGoodSmall color="green" />
+          <Text size="large">Power On initiated for 1 device.</Text>
+        </Box>
+      )}
+      {!isLoading && !isSuccessful && (
+        <Text size="large">FTCUSAMountain_ML350Gen10_testcluster</Text>
+      )}
+      {isLoading && (
+        <Spinner
+          alignSelf="center"
+          size="medium"
+          message={{
+            start: 'Loading more data.',
+            end: 'Data has been loaded.',
+          }}
+        />
+      )}
+      {!isSuccessful && (
+        <Box
+          justify="end"
+          gap="medium"
+          margin={{ top: 'medium', bottom: 'small' }}
+          direction="row"
+        >
+          <Button label="Cancel" />
+          <Button
+            label="Power on 1 Device"
+            onClick={PowerOnLoading}
+            primary
+            type="submit"
+          />
+        </Box>
+      )}
+      {isSuccessful && (
+        <Box width="large" gap="medium" justify="center" pad="small">
+          <Heading level={4} size="small" margin="none">
+            Feedback on powering devices
+          </Heading>
+          <Form
+            method="post"
+            validate="submit"
+            kind="survey"
+            onChange={(nextValue, { touched }) => {
+              setRating(touched);
+            }}
+            onSubmit={value => {
+              console.log('submit', value.value);
+            }}
+          >
+            <Box gap="medium">
+              <FormField
+                contentProps={{
+                  border: false,
+                }}
+                label="How easy was it for you to power off your selected devices?"
+                htmlFor="power-rating"
+                name="power-rating-example"
+              >
+                <StarRating id="power-rating" name="power-rating-example" />
+              </FormField>
+              {rating && (
+                <Box gap="medium">
+                  <FormField
+                    name="power-textArea-example"
+                    htmlFor="power-text-area-example"
+                    label="How satisfied were you with this process?"
+                  >
+                    <TextArea
+                      name="power-textArea-example"
+                      id="power-text-area-example"
+                      placeholder="i.e. ideas, inspirations, or concerns"
+                    />
+                  </FormField>
+                  <Box align="end" justify="end">
+                    <Button
```

Comment:
It feels like we are trapping the user and forcing them to complete the survey. They have no choice here. 

I think the feedback should be presented in a subsequent frame (i.e. following the last step of the flow.

![Screenshot 2023-01-06 at 4 28 33 PM](https://user-images.githubusercontent.com/1756948/211116976-825567c4-6f6e-466d-b3b4-45f31f1ad75f.png)